### DISCUSSION TITLE
Como ativar o modo de seleção visual em bloco no Lazyvim

### DISCUSSION THREAD

**Discussion by thiagoirenio** [LazyVim/LazyVim#4318] [Q&A]
Como ativar o modo de seleção visual em bloco no Lazyvim?

**Comment by rogeriofrsouza** [comment]
Comando `C-v`

https://neovim.io/doc/user/visual.html

**Comment by thiagoirenio** [comment]
In the case of my lazyvim the 'C-v' command comes with paste natively, there is no other native key mapping for it.

**Reply by rogeriofrsouza** [reply]
Are you on Windows?
Try using `C-q` instead.
https://stackoverflow.com/a/3736856/19551811
